pos_accounting/src/pool/storage/helpers.rs

```diff
@@ -0,0 +1,43 @@
+use crate::{error::Error, storage::PoSAccountingStorageWrite};
+use chainstate_types::storage_result;
+
+pub struct StorageTempAccessor<'a, S, Getter, Setter, Deleter> {
```

Comment:
Please avoid using "Temp" in names. In theory, everything is temporary.

I would call this StorageAmountController, or StorageAmountWrapper.

Reply:
If you're not using this just for amounts, you can call this StorageValueController, or StorageValueWrapper

Reply:
What if I rename it to `BorrowedStorage`? Imo Accessor, Wrapper or Controller are too abstract. This struct does nothing but borrows and was introduced to please borrow checker.

Reply:
BorrowedStorageValue, maybe?